test/Integration/GetWorksByPublicKey.ts

```diff
@@ -122,4 +122,12 @@ export class GetWorksByPublicKey {
       Expect(claims[i].dateCreated.toISOString()).toBe(expectedClaims[i].dateCreated.toISOString())
     }
   }
+
+  @AsyncTest()
+  @TestCase('')
+  async getWorks422(publicKey: string) {
```

Comment:
Can you rename this test to something like `getWorksShouldFailWith422WhenPassingAnInvalidArgument`?

Reply:
Yeah, I like it!